site/pages/cheatsheets.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+{% block title %} Cheatsheets {% endblock %}
+{% block meta_description %} Cheatsheets {% endblock %}
```

Comment:
I've since learned that Cheatsheet is not one word, Should probably be Cheat Sheets. Idk if you care about this, I don't think it matters much but while I was reviewing figured I'd mention it.